src/models/parameters/training/ultralytics/ultralytics_hyper_parameters.py

```diff
@@ -0,0 +1,22 @@
+from src.models.parameters.common.hyper_parameters import HyperParameters
+
+from picsellia.types.schemas import LogDataType
+
+
+class UltralyticsHyperParameters(HyperParameters):
+    def __init__(self, log_data: LogDataType):
+        super().__init__(log_data=log_data)
+
+        self.device = self.extract_parameter(
+            keys=["device"], expected_type=str, default="cpu"
+        )
+        self.use_cache = self.extract_parameter(
+            keys=["cache", "use_cache"],
+            expected_type=bool,
+            default=False,
+        )
+        self.save_period = self.extract_parameter(
+            keys=["save_period"],
+            expected_type=int,
+            default=-1,
+        )
```

Comment:
I think we should also add the following parameters:
- patience
- optimizer
- cos_lr
- close_mosaic (set to 0 by default)
- amp
- warmup_epoch
- warmup_momentum
- warmup_bias_lr
- overlap_mask
- mask_ratio

WDYT ?